NEWS.md

```diff
@@ -143,6 +23,6 @@ of syslog-ng, contribute.
 We would like to thank the following people for their contribution:
 
 Andras Mitzki, Antal Nemes, Attila Szakacs, Balazs Scheidler,
-Balázs Barkó, Benedek Cserhati, Gabor Nagy, L4rS6, Laszlo Budai, Laszlo Szemere,
-LittleFish33, László Várady, Norbert Takacs, Peter Czanik, Peter Kokai,
-Todd C. Miller, Tomáš Mózes, Zoltan Pallagi
+Gabor Nagy, Josef Schlehofer, Laszlo Budai, Laszlo Szemere,
```

Comment:
Can we add  Benedek Cserhati, Balázs Barkó and Szilárd Parrag manually to the news file for now, please?
The script is no up-to-date.
I`ve created a PR for that #3737.

Reply:
Thank You for the comment and the related PR! I fixed the list (and made sure, Josef Schlehofer will remain in place).

I don't want to resolve the conversation by myself. Could you please close it, as a double check?

Reply:
Checked, it's perfect. Thanks!